examples/serialize.rs

```diff
@@ -0,0 +1,18 @@
+use rdev::{Event, EventType, Key};
+use serde_json;
+use std::time::SystemTime;
+
+fn main() {
+    let event = Event {
+        event_type: EventType::KeyPress(Key::KeyS),
+        time: SystemTime::now(),
+        name: Some(String::from("S")),
+    };
+
+    let serialized = serde_json::to_string(&event).unwrap();
+
+    let deserialized: Event = serde_json::from_str(&serialized).unwrap();
+
+    println!("Serialized event {:?}", serialized);
```

Comment:
If we implement `PartialEq` and `Eq` on `Event`, we could `assert_eq!(event, deserialized)` here.